index.html

```diff
@@ -22,7 +22,54 @@
         <!--"header end"-->
     
         <!--"main-block start"-->
-        <section id="main-block"></section>
+        <section id="main-block">
+            <div class="backgroundContainer" id="main-block">
```

Comment:
Class names aren't named properly (backgroundContainer instead of background-container)